README.md

```diff
@@ -12,20 +12,24 @@ User requirements
 
 2. Carbonite console 8.3.0.293+
 
-3. Carbonite Migrate — The source server must have Carbonite Migrate installed and licensed on it.
+3. Carbonite Migrate — the machine you will be migrating (source) **must** have Carbonite Migrate software installed and licensed on it. This machine also needs to be added to the Carbonite console application on your Carbonite server:
+![alt](https://s3.amazonaws.com/device42/carbonite-migrate/mstsc_Fv0eg6nSnA.png)
 
-4. PowerShell 5.1: if you are using Windows 10, you already have the necessary
+4. Carbonite Target software must be installed on the replication server and the server added to the Carbonite console application. This machine will be called the "Carbonite target" in the script's questions.
```

Comment:
Uppercase console.  Just to confirm, Carbonite Console is the name of the product?

Reply:
Change to Carbonite Replication Console.